forge-game/src/main/java/forge/game/Match.java

```diff
@@ -288,6 +288,7 @@ private void prepareAllZones(final Game game) {
             }
 
             Deck myDeck = psc.getDeck();
+            player.setDraftNotes(myDeck.getDraftNotes());
```

Comment:
Do we need to duplicate this data if we can just navigate to the Deck object anyway?

Reply:
EHh.. the Player object doesn't really have great access to the Deck object. It gets denormalized down into PlayerZones and such. I think its just simpler to send it down.